x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/flyout_home.tsx

```diff
@@ -346,50 +351,53 @@ const CreateJobTab = (props: CreateJobTab) => {
             }
           />
         </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiCard
-            isDisabled={!props.hasSetupCapabilities}
-            icon={<EuiIcon type={'logoKubernetes'} size="xl" />}
-            title={
-              <FormattedMessage
-                defaultMessage="Kubernetes Pods"
-                id="xpack.infra.ml.anomalyFlyout.create.k8sTitle"
-              />
-            }
-            description={
-              <FormattedMessage
-                defaultMessage="Detect anomalies for memory usage and network traffic on Kubernetes Pods."
-                id="xpack.infra.ml.anomalyFlyout.create.k8sDescription"
-              />
-            }
-            footer={
-              <>
-                {props.hasK8sJobs && (
-                  <EuiButtonEmpty
-                    data-test-subj="infraCreateJobTabRecreateJobsButton"
-                    onClick={props.createK8s}
-                  >
-                    <FormattedMessage
-                      defaultMessage="Recreate jobs"
-                      id="xpack.infra.ml.anomalyFlyout.create.recreateButton"
-                    />
-                  </EuiButtonEmpty>
-                )}
-                {!props.hasK8sJobs && (
-                  <EuiButton
-                    data-test-subj="infraCreateJobTabEnableButton"
-                    onClick={props.createK8s}
-                  >
-                    <FormattedMessage
-                      defaultMessage="Enable"
-                      id="xpack.infra.ml.anomalyFlyout.create.createButton"
-                    />
-                  </EuiButton>
-                )}
-              </>
-            }
-          />
-        </EuiFlexItem>
+        {!props.hideJobType && (
+          <EuiFlexItem>
+            <EuiCard
+              data-test-subj="infraK8sJobCard"
+              isDisabled={!props.hasSetupCapabilities}
+              icon={<EuiIcon type={'logoKubernetes'} size="xl" />}
+              title={
+                <FormattedMessage
+                  defaultMessage="Kubernetes Pods"
+                  id="xpack.infra.ml.anomalyFlyout.create.k8sTitle"
+                />
+              }
+              description={
+                <FormattedMessage
+                  defaultMessage="Detect anomalies for memory usage and network traffic on Kubernetes Pods."
+                  id="xpack.infra.ml.anomalyFlyout.create.k8sDescription"
+                />
+              }
+              footer={
+                <>
+                  {props.hasK8sJobs && (
```

Comment:
nit: this condition is repeating right? We can maybe put both buttons under the same condition 🤔

Reply:
It's not the same condition and I didn't touch that, one is for **creating** jobs and the other one it's for **recreating** them

Reply:
I see now, I missed that when comparing the diff, thanks